src/core/FuseBox.ts

```diff
@@ -228,21 +228,13 @@ export class FuseBox {
             this.context.filterFile = opts.filterFile;
         }
 
-        if (opts.log) {
-            if (typeof opts.log === "boolean") {
-                this.context.doLog = opts.log;
-            }
-
-            if (typeof opts.log === "object" && opts.log.enabled) {
-                this.context.doLog = opts.log.enabled;
-                this.context.log.printLog = opts.log.enabled;
-                this.context.log.showBundledFiles = opts.log.showBundledFiles;
-            }
+        if (typeof opts.log === "boolean") {
+            this.context.log.printLog = opts.log;
         }
 
-        if (opts.log !== undefined) {
-            this.context.doLog = true;
```

Comment:
What happened to doLog? I see it's been removed.

Reply:
It's on true already https://github.com/sod/fuse-box/blob/master/src/core/WorkflowContext.ts#L155

And was never really user overwritable because this line just set it back to true.